doc/man3/OBJ_nid2obj.pod

```diff
@@ -139,6 +139,8 @@ These functions cannot return B<const> because an B<ASN1_OBJECT> can
 represent both an internal, constant, OID and a dynamically-created one.
 The latter cannot be constant because it needs to be freed after use.
 
+These functions were not thread safe in OpenSSL 3.0 and before.
```

Comment:
This sentence is contradictory with the next change below, no ?

Reply:
Correct.  I'll remove the history entry.